site/pages/create-consequence-network/[disruptionId]/[consequenceIndex].page.tsx

```diff
@@ -113,7 +113,7 @@ const CreateConsequenceNetwork = (props: CreateConsequenceNetworkProps): ReactEl
                                     props.disruptionSummary ? stateUpdater(props.disruptionSummary, "description") : "";
```

Comment:
Can we change the name of this prop to disruptionDescription to reflect it is copy the description not a summary

Reply:
done